uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +70,22 @@ protected function cleanup() {
     }
   }
 
+  private function loadRequiredData() {
```

Comment:
I was running tests for another extension that clear data from some tables. I don't think we should assume that anything will be in the database that's not added the the extension installer.